app/backend/src/profile/profile.gateway.spec.ts

```diff
@@ -3,13 +3,20 @@ import { ProfileGateway } from "./profile.gateway";
 import { ProfileService } from "./profile.service";
 import { PrismaService } from "src/prisma/prisma.service";
 import { ConfigService } from "@nestjs/config";
+import { TokenStorageService } from "src/tokenstorage/token-storage.service";
 
 describe("ProfileGateway", () => {
   let gateway: ProfileGateway;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [ProfileGateway, ProfileService, PrismaService, ConfigService]
+      providers: [
+        ProfileGateway,
+        ProfileService,
+        PrismaService,
+        ConfigService,
+        TokenStorageService
```

Comment:
Again, why does the profile module need to access the token storage?

Reply:
yes , for tests otherwise the testers don't work